### PR TITLE
now puts user shows the role of the user

### PR DESCRIPTION
**Pull request by AdamC12**

### CODE REVIEW

File: project.rb [hunk 1]
```diff
@@ -16,8 +17,18 @@ def remove_task(task_name)
...
+  end
+
+  def to_s(user)
     task_list = @tasks.map(&:to_s).join("\n")
-    "Project: #{@name}\nTasks:\n#{task_list}"
+    "Project: #{@name}\nRole: #{user_role(user)}\nTasks:\n#{task_list}"
+  end
+
+  def user_role(user)
+    member = @team.select { |member| member[:user] == user.name }
```
Comment (Robfaldo) [inline project.rb:31]: if you do `.find` instead of `.select` it will return the first one that matches (or `nil` if none match)

File: user.rb [hunk 2]
```diff
@@ -8,7 +9,8 @@ def initialize(name)
     @projects = []
   end
 
-  def add_project(project)
+  def add_project(project, user, role)
```
Comment (Robfaldo) [inline user.rb]: i don't think you need to pass a user to this method, given you're already in the user class

File: project.rb (hunk 1 above)
Comment (Robfaldo) [reply project.rb:31]: optional - just a nitpick

File: user.rb (hunk 2 above)
Comment (AdamC12) [reply user.rb]: should i use self instead?

Comment (Robfaldo) [reply user.rb]: yeah exactly

Comment (AdamC12) [reply user.rb]: i updated this :)

**Pull request merged**